datasets/br/bcb_disqualified_persons/crawler.py

```diff
@@ -0,0 +1,90 @@
+from typing import List
+from zavod import Context, helpers as h
+
+
+def fetch_data(context: Context) -> List[dict]:
+    """
+    Fetches data from the website, or raises an exception on failure.
+
+    :param context: The context object.
+    :return: List of dicts containing data.
+    """
+    response = context.fetch_json(context.data_url)
+    if "value" not in response:
+        context.log.error("Value not found in JSON")
+        return []
+    return response["value"]
+
+
+def create_entity(input_dict: dict, context: Context):
+    """
+    Creates an entity from the raw data.
+
+    :param input_dict: Data dict extracted from source.
+                       Should have at least following keys:
+                        - CPF
+                        - Nome
+    :param context: The context object.
+    """
+    entity = context.make("Person")
+    tax_number = input_dict["CPF"]
+    entity.id = context.make_id(tax_number)
+    entity.add("name", input_dict["Nome"])
+    entity.add("taxNumber", tax_number)
+    entity.add("country", "br")
+    entity.add('topics', 'corp.disqual')
+
+    return entity
+
+
+def create_sanction(input_dict: dict, entity, context: Context):
+    """
+    Creates the sanction for a given entity and the raw data.
+
+    :param input_dict: Data dict extracted from source.
+                       Should have at least following keys:
+                        - PAS
+                        - Prazo_em_ano
+                        - Inicio_do_cumprimento
+                        - Prazo_final_penalidade
+    :param entity: Entity the sanction refers to.
+    :param context: The context object.
+    """
+
+    sanction = h.make_sanction(context, entity)
+    sanction.add(
+        "program", "Brazil's Central Bank General Register of Disqualified Persons"
+    )
+    sanction.add("authority", "Brazil's Central Bank")
+
+    # The ID of the process
+    sanction.add(
+        "description",
+        "Administrative Sanctioning Process Number: {}".format(input_dict.pop("PAS")),
+    )
+
+    # The duration is always in years
+    sanction.add("duration", "{} year(s)".format(input_dict.pop("Prazo_em_anos")))
+
+    # The start and end dates are in the format YYYY-MM-DD
+    sanction.add("startDate", input_dict.pop("Inicio_do_cumprimento"))
+    sanction.add("endDate", input_dict.pop("Prazo_final_penalidade"))
+
+    return sanction
```

Comment:
Could you add Penalidade under Sanction:provisions with the three-letter language code for bazil?

Reply:
I think I didn't fully understood what you mean here, do you want to add something like:

```
sanction.add("provisions", "PNL")
```

Where PNL is an acronym for "Penalidade"?

Reply:
ah no, sorry - I mean
```
sanction.add("provisions", input_dict.pop("Penalidade"), lang="por") - right now the value is always INABILITAÇÃO which I'm not sure google translate is translating correctly to "disability", but it seems to fit the "Scope of sanctions", right?

Reply:
It's always INABILITAÇÃO because this is the "Quadro Geral de Inabilitados". 

The word inabilitação comes from the word habilitação (it comes from the same latin root for able) which I think would be best translated to license. So for instance Driver's License is "Habilitação para dirigir" (License to drive). The "in-" prefix means the negative, so in-abilitação means they "lost their license" (more in a figurative way) and they are not able to hold some positions in financial companies.

Does that mean the same as disability? I'm not sure what would be the correct English term for that.